docs/for-devs/get-started/lab-1.md

```diff
@@ -22,7 +22,7 @@ We assume you (or the platform administrator) have:
 - Trivy Operator
 - Gatekeeper (with option `Disable validating webhook` set to true)
 
-For the [Use OpenTelemery](lab-27) Lab, the Tempo app needs to be enabled together with tracing in the `Istio` and `Nginx Ingress` apps.
+For the [Use OpenTelemery](lab-27.md) Lab, the Tempo app needs to be enabled together with tracing in the `Istio` and `Nginx Ingress` apps.
 
 3. Created a team called `demo` with `Managed prometheus and alert manager` enabled
 4. [Created an account](/docs/apps/keycloak#create-a-user-in-keycloak) and added your user to the team group in Keycloak
```

Comment:
what about this one?

Reply:
the `#create-a-user-in-keycloak` ? For some reason things with a refference in the page with # don't need it